root/search/components/RecordingResults.js

```diff
@@ -125,7 +125,7 @@ export const RecordingResultsInline = ({
         <th>{l('Artist')}</th>
         <th>{l('ISRCs')}</th>
         <th>{l('Release')}</th>
-        {$c && $c.session && $c.session.tport ? <th>{l('Tagger')}</th> : null}
+        {$c?.session?.tport ? <th>{l('Tagger')}</th> : null}
```

Comment:
I cannot understand why `$c` is optional here.
It has been made optional with commit 3e0afa96b7a1626f2fcf5c33e705c6c7a4e771ef but I don’t understand that change either.

Reply:
@reosarevok any idea? :) you authored 3e0afa9

Reply:
I made it optional because it was unused in, say, ArtistResultsInline. Is that not the right way to do this? Should we have different prop types for with and without context?

Reply:
Ah okay. Perhaps, since we do that for `ResultsPropsT` / `ResultsPropsWithContextT` already. I'll leave it for a different PR though.